src/main/resources/application.yaml

```diff
@@ -12,6 +12,10 @@ spring:
 
 bulk-scan-processor-url: ${BULK_SCAN_PROCESSOR_URL:}
 
+queue:
+  conn-string: ${QUEUE_CONNECTION_STRING:}
```

Comment:
Is terraform config part of separate PR ?

Reply:
it's actually already on master, thanks to @timja 
https://github.com/hmcts/bulk-scan-orchestrator/blob/master/infrastructure/main.tf#L39